.github/workflows/backport-on-comment.yml

```diff
@@ -0,0 +1,20 @@
+name: Backport labeled PRs triggered by comment
+on:
+  issue_comment:
+    types: [ created, edited ]
```

Comment:
Do you want to trigger this also for "edited"?

Reply:
Probably not a good idea. Thanks for pointing it out